spec/integration/federation/receive_federation_messages.rb

```diff
@@ -0,0 +1,224 @@
+require "spec_helper"
+require File.dirname(__FILE__) + "/factories"
```

Comment:
Rather than this, place the factories into the correct folder, `spec/support/federation_integration_factories.rb` or something.